docs/src/main/paradox/scala/http/client-side/websocket-support.md

```diff
@@ -25,33 +25,44 @@ when using WebSockets for one-way communication may not be what you would expect
 ## Message
 
 Messages sent and received over a WebSocket can be either `TextMessage` s or `BinaryMessage` s and each
-of those has two subtypes `Strict` or `Streamed`. In typical applications messages will be `Strict` as
+of those has two subtypes `Strict` (all data in one chunk) or `Streamed`. In typical applications messages will be `Strict` as
```

Comment:
It was there on the in the Java version, and I liked it so much that I kept it :smiley: